src/ontology/mondo-edit.obo

```diff
@@ -535349,7 +535351,7 @@ xref: OMIM:619059 {source="MONDO:equivalentTo"}
 xref: UMLS:C5436712 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:1773430"}
 is_a: MONDO:0003847 {source="https://orcid.org/0000-0001-5208-3432"} ! hereditary disease
 is_a: MONDO:0033885 {source="OMIM:619059"} ! mitochondrial complex IV deficiency, nuclear-type
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/2294 {source="OMIM:619059"} ! COX8A
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/2294 ! COX8A
```

Comment:
The gene association is marked as provisional, phenotype name starts with ?, https://omim.org/entry/619059, and therefore does not meet the criteria to be maintained by the omim ingest pipeline. Please confirm if this needs to be maintained and added to the "protected" file.

Reply:
This source should be maintained and added to the "protected" file